CHANGELOG.md

```diff
@@ -7,9 +7,11 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 ## [2.X.X] - 20XX-XX-XX
 ### Added
 - subsector, has subsector, subsector of (#1788)
+- industrial technology, industrial plant (#1840)
 
 ### Changed
 - electricity sector, industry sector, CRF sector (IPCC 2006) individuals (#1788)
+- electricity import/export value (#1864)
```

Comment:
While resolving merge conflicts in CHANGELOG.md for this PR I noticed that in PR #1864 I put `- electricity import/export value (#1864)` in the wrong part of CHANGELOG.md. I fixed this on the fly.